### PR TITLE
Update common-measuring-vibration.rst

### PR DESCRIPTION
**Pull request by kd0aij**
update for addition of core instance number to NKF1 log message and removal of NKF6
TODO: update vibration-measuring-leans.png

### CODE REVIEW

File: common/source/docs/common-measuring-vibration.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ find they are out-of-tolerance then follow the advice found on the :ref:`Vibrati
 
 Real-Time view in Ground Station
 --------------------------------
-
+nkf
```
Comment (rmackay9) [inline common/source/docs/common-measuring-vibration.rst]: This "nkf" looks like it might have been an accidental addition..

Comment (kd0aij) [reply common/source/docs/common-measuring-vibration.rst]: yep

**Comment by kd0aij** [comment]
@Hwurzburg Is anyone else working on updating references like this across the wiki? I'll do the rest of the textual changes if nobody else is working on it.

**Comment by Hwurzburg** [comment]
@kd0aij Mark, you da man! no one else working on it...please help appreciated!thanks

**Comment by kd0aij** [comment]
@Hwurzburg @rmackay9  I think I fixed all the existing text, but fixing the images would require a bit more work.

**Comment by rmackay9** [comment]
Merged, thanks!

updating the images would be great but this is still an improvement

**Pull request merged**